charts/sonarqube/templates/secret.yaml

```diff
@@ -1,4 +1,5 @@
-{{- if eq (include "postgresql.useInternalSecret" .) "true" -}}
+---
+{{- if not (or (or .Values.postgresql.enabled .Values.postgresql.existingSecret) .Values.jdbcOverwrite.jdbcSecretName)}}
```

Comment:
couldn't this expression be simplified to 
`{{- if not (or .Values.postgresql.enabled .Values.postgresql.existingSecret .Values.jdbcOverwrite.jdbcSecretName)}}` ?

Reply:
Unless Helm supports this without documenting it I was under the impression `or` would only support 2 argument expressions. [Helm Docs](https://helm.sh/docs/chart_template_guide/function_list/#or)
Besides that this suboptimal logic construct exists just to keep 'backwards compatibility' to allow the 'deprecation' of `postgress.existingSecret`. If you would be fine with with it I could remove `postgress.existingSecret` entirely and simplify the code a bit (especially give that this likely wont be a patch version)

Reply:
okay I had another read up on go templates and it seems your proposal is indeed supported and working (tested by `helm template .`)

:man_facepalming: I should have tried this way earlier xD